packaging/rpm/libnats.spec

```diff
@@ -0,0 +1,55 @@
+%global source_name nats.c
+%define debug_package %{nil}
+
+Name:          libnats
+Version:       3.1.1
```

Comment:
We are currently at version 3.2.0. Also, would be nice if that could be updated when generating version, otherwise it will need to be manually updated before I do a new release..